packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -101,6 +107,12 @@ export default class Dropdown extends React.Component<Props, State> {
         };
     }
 
+    componentDidMount() {
+        if (this.props.opened === true || this.props.opened === false) {
```

Comment:
Is there any particular reason to evaluate opened here? Based on my initial observation, it seems you could remove this condition, but I want to double check with you first.

Reply:
Could this just be `this.props.opened != null`?

Reply:
@jandrade I wanted to override the opened state only if opened was passed in, I think @somewhatabstract’s suggestion works the same